### ISSUE TITLE
Web ui needs to be refreshed before it shows the products

### ISSUE THREAD

**Issue by siamaksade** [jbossdemocentral/coolstore-microservice#24]

**Comment by siamaksade** [comment]
@jamesfalkner @tqvarnst

**Comment by siamaksade** [comment]
After showing the products, still more refreshes needed to show the correct product inventory

**Comment by siamaksade** [comment]
It's also in master branch. I used the provisioning script to deploy the demo

**Comment by siamaksade** [comment]
This issue happens deterministically on any of the stable branches as well as in RHPDS.

**Comment by tqvarnst** [comment]
I believe I've identified the issue for this. The problem is that the first request takes longer then normal and triggers the circuit breaker. The default timeout is 1000 ms which is kind of low, but it's hard to know exactly what to set the timeout to. I've decided to go for the the following values.

`
.hystrixConfiguration()                      
    .executionTimeoutInMilliseconds(5000).circuitBreakerSleepWindowInMilliseconds(10000)
.end()
`
If this issue occurs again we might have to trim this value.

I will close this issue when I issue the pull request.
